### ISSUE TITLE
is it possible to Mount a folder path as a drive?

### ISSUE THREAD

**Issue by starinspace** [joncampbell123/dosbox-x#2651]
I have created a image with win95 installation, when i want to install games I have to create a new image, mount that in windows 10 and add the files i want before i can mount it as a new hard drive like d: in dosbox-x.

But....is there a way to mount a path/folder instead of an image file? Or adding this feature?

Or. Alternative question, is it possible to reach a special folder through "internet connection" inside windows 95 to get to a folder to be able to reach files?

**Comment by Wengier** [comment]
Right now guest systems can only see disk or CD images, not host paths, but you can swap images files dynamically. Moreover, I believe it is possible to set up network connections for Windows 95 to access network paths, provided that NE2000 networking is enabled. More information can be found in the Wiki page.

**Comment by rderooy** [comment]
Yes if you configure NE2000 properly (with the pcap backend in Win10), you should be able to mount a network share. However, this will apparently not work in combination with a modern Win10 as file server. You will need to use a SMB fileserver that still  supports older SMB 1.x protocol versions.

See the wiki for how to configure networking in DOSBox-X for more details.
